### ISSUE TITLE
相册在博客中的显示异常

### ISSUE THREAD

**Issue by Orionxty** [BNDong/Cnblogs-Theme-SimpleMemory#392]
只有第一个相册的内容能在博客中显示, 而其他的都显示不出来

**Comment by BNDong** [comment]
相册没适配过，我这两天看下

**Comment by BNDong** [comment]
博客园原生功能支持一个页面展示多个相册？我没有找到

**Comment by BNDong** [comment]
加功能这肯定做不到，相册的页面的效果可以优化下

**Comment by Orionxty** [comment]
对, 就是我点https://www.cnblogs.com/Usky/category/2315677.html, 然后不会直接显示该相册的图片, 需要在界面中再点击选择要打开的相册

**Comment by wangyang0210** [comment]
<img width="1502" alt="image" src="https://github.com/BNDong/Cnblogs-Theme-SimpleMemory/assets/36377605/c1a3ece9-019c-48a3-8155-184bca3cac90">
